internal/core/unittest/test_expr.cpp

```diff
@@ -1413,9 +1413,9 @@ TEST(Expr, TestExprs) {
                           int n) -> std::shared_ptr<query::Expr> {
         switch (test_type) {
             case UnaryRangeExpr:
-                return std::make_shared<query::UnaryRangeExprImpl<int8_t>>(
-                    ColumnInfo(int8_fid, DataType::INT8),
-                    proto::plan::OpType::GreaterThan,
+                return std::make_shared<query::UnaryRangeExprImpl<int64_t>>(
```

Comment:
why change the OpType?
add a new testcase ?

Reply:
test type, can ignore